Branch-SDK/Branch-SDK/Requests/BranchShortUrlSyncRequest.m

```diff
@@ -84,7 +84,9 @@ - (NSString *)createLongUrlForUserUrl:(NSString *)userUrl {
 }
 
 + (NSString *)createLinkFromBranchKey:(NSString *)branchKey tags:(NSArray *)tags alias:(NSString *)alias type:(BranchLinkType)type matchDuration:(NSInteger)duration channel:(NSString *)channel feature:(NSString *)feature stage:(NSString *)stage params:(NSDictionary *)params {
-    NSMutableString *baseUrl = [[NSMutableString alloc] initWithFormat:@"%@/a/%@?", BNC_LINK_URL, branchKey];
+    BNCPreferenceHelper *preferenceHelper = [BNCPreferenceHelper preferenceHelper];
+    NSMutableString *baseUrl = [preferenceHelper.userUrl mutableCopy];
```

Comment:
This presumes the SDK has been initialized once and that userUrl has been filled. Why change?